CONTRIBUTING.md

```diff
@@ -23,6 +23,15 @@ compiled code in `dist`.
 
 Test cases are declared in `src/test/*.spec.ts`, and test fixtures live in `./tests`.  They can be run with `npm run test-local`.
 
+To run a subset of tests:
+
+```
+# Use ava's --match flag to match the name of a test or suite
+# https://github.com/avajs/ava/blob/main/docs/05-command-line.md
+# Don't forget the * wildcards
+npm run test-local --match '*esm loader*'
```

Comment:
This didn't work for me (on MacOS > zsh). I needed an extra `--`:
```sh
npm run test-local -- --match '*path mapping esm*'
```

Reply:
Good catch; fixed.